lesson-3/README.md

```diff
@@ -226,37 +180,192 @@ spec:
   repository: "docker.io/prgcont/etcd"
 ```
 
-Note: You need to update RBAC rules if you want ETCD Operator to manage resources across all kubernetes cluster. 
+Note: You need to update RBAC rules if you want etcd operator to manage resources across all kubernetes cluster. 
 
-### Cleanup ETCD Operator from k8s cluster
 
-```bash
-kubectl delete etcdcluster example-etcd-cluster
-kubectl delete -f example/deployment.yaml
-kubectl delete endpoints etcd-operator
-kubectl delete crd etcdclusters.etcd.database.coreos.com
-kubectl delete clusterrole etcd-operator
-kubectl delete clusterrolebinding etcd-operator
+
+
+## Write simple dummy Operator in Python
+
+We will try to create a very simple 'operator' in Python. IT will be able to schedule desired
+count of replicas of our Gordon up and it will be able to assure that desired count of replicas
```

Comment:
This sentense is very unclear about real actions:
* Is it going to be a scheduler (to schedule)?
* Is it going to be controller-manager (to ensure replicas are running)?

What about It will be creating pods for Gordon according to our CRD?